client/app/pages/dashboards/dashboard.html

```diff
@@ -96,8 +96,8 @@ <h3>
       ng-class="{'preview-mode': !$ctrl.layoutEditing, 'editing-mode': $ctrl.layoutEditing}">
       <div class="dashboard-widget-wrapper"
         ng-repeat="widget in $ctrl.dashboard.widgets track by widget.id"
-        gridstack-item="widget.options.position" gridstack-item-id="{{ widget.id }}">
-        <div class="grid-stack-item-content" data-test="WidgetId{{ widget.id }}">
+        gridstack-item="widget.options.position" gridstack-item-id="{{ widget.id }}"  data-test="WidgetId{{ widget.id }}">
+        <div class="grid-stack-item-content">
```

Comment:
Moved to parent node in order to locate the handle (which is a sibling)

https://github.com/getredash/redash/blob/689e1a63bff22e5e7cdc354890e5c576cd307ea1/cypress/integration/dashboard/dashboard_spec.js#L112-L116